### DISCUSSION TITLE
Is there any documentation or guidance for me to run the test scripts in the tests folder?

### DISCUSSION THREAD

**Discussion by aaa147147** [openthread/openthread#11079] [Q&A] [answered]
**How should I get started to use the following test folder.**
[[openthread tests](https://github.com/openthread/openthread/tree/main/tests)](https://github.com/openthread/openthread/tree/main/tests)

Is there an official website or document that can help me? I haven't found it yet

**Comment by zhanglongxia** [comment]
You can run the command [./script/test](https://github.com/openthread/openthread/blob/main/script/test#L496) to show the usage of running tests under `tests` folder.

**Comment by superwhd** [comment, marked as answer]
You can also refer to how tests are executed in GitHub workflows: https://github.com/openthread/openthread/tree/main/.github/workflows. You should be able to run them in the same way locally following the workflow steps.
